modules/ROOT/edits/download-and-install-service-mesh.adoc

```diff
@@ -0,0 +1,85 @@
+
+To download the Anypoint Service Mesh client, type the following command at the command prompt:
+
+`curl -Ls http://anypoint.mulesoft.com/servicemesh/xapi/v1/install > service-mesh && chmod +x service-mesh`
+
+The Anypoint Service Mesh client is now downloaded on your file system.
+
+== Install Anypoint Service Mesh
+
+When you invoke the installer, Istio, Service Catalog, Anypoint Service Mesh Broker, and ClusterIP Service components are installed in your Kubernetes cluster. The installer performs several steps as part of the installation, such as checking for dependencies and performing various configurations. 
+
+=== Install without Proxy
+To install Anypoint Service Mesh, perform the following steps:
+
+. In the location from which you downloaded your Anypoint Service Mesh client, type:
++
+`./service-mesh install`
++
+The `Welcome to Anypoint Service Mesh installer` message is displayed.
+. Enter your client ID that you obtained as part of the prerequisites.
+. Enter your client secret that you obtained as part of the prerequisites.
+. Enter the path where your Anypoint Service Mesh license is stored.
++
+Anypoint Service Mesh installs within few minutes and displays a status: 
++
+image::service-mesh-installer-output.png[Example Installation Output]
++
+If an installation error occurs, the installer times out and uninstalls all previously installed Anypoint Service Mesh components.
+. Verify the status of all pods:
++
+`kubectl get pods -n service-mesh`
++
+Anypoint Service Mesh is installed and a status of all pods is displayed:
++
+image::verified-service-mesh-installer-output.png[]
+
+=== Install with Proxy
+
+If you need to install Anypoint Service Mesh in a proxy environment, provide the proxy configuration during the installation. 
+
+. In the location from which you downloaded your Anypoint Service Mesh client, type:
++
+`./service-mesh install --clientId=<clientId> --clientSecret=<clientSecret> --license=<license absolute or relative path> --platformUri=<Anypoint Platform Uri> --proxyHost=<host> --proxyProtocol=<protocol> --proxyPort=<port> --proxyUser=<proxy username> --proxyPassword=<proxy password>`
++
+[%header%autowidth.spread]
+|===
+| Parameter Name | Description | Required/Optional | Default Value
+| `clientId` | The client ID of the environment (recommended) or the organization. Use the client ID that you obtained as part of performing the permission and roles prerequisites. | Required | None
+| `clientSecret` | The client secret of the environment (recommended) or the organization. Use the client secret that you obtained as part of performing the permission and roles prerequisites.| Required | None
+| `license` | The Anypoint Service Mesh license you obtained as part of the prerequisites. Provide the path to the location where you license is stored. | Required | None
+| `platformUri` | The URL of the proxy.| Optional | https://anypoint.mulesoft.com
+| `proxyHost` | The name of the proxy host. | Optional | None
```

Comment:
```suggestion
| `proxyHost` | The name of the proxy host | Optional | None
```
GLOBALLY remove periods from the end of table column descriptions that are not complete sentences.
Try to make column entries parallel. If even one MUST be a sentence, then all of them must end with periods.

Reply:
I will stop removing the periods for you, to cut down on the number of review comments. Please go through and fix.